app/instrumentation-tests/src/org/commcare/utils/InstrumentationUtility.kt

```diff
@@ -310,6 +321,21 @@ object InstrumentationUtility {
         }
     }
 
+    /**
+     * A utility to hide the keyboard
+     */
+    fun hideKeyboard(){
+        onView(ViewMatchers.isRoot()).perform(ViewActions.closeSoftKeyboard())
+    }
+
+    fun verifyIntentText(text : String){
```

Comment:
this function is better suited in the test class itself instead of this utility class as I don't think there are much chances of this getting used by another test.

Reply:
removed the function and added the steps in the function